### DISCUSSION TITLE
node build.js is not working.

### DISCUSSION THREAD

**Discussion by dilankamr** [fabricjs/fabric.js#8251] [General]
Hi, 

I have customized this repo by forking. The developers have migrated src files from .js to .ts. After changing files from .js to .ts, node build.js is not working. After pulling new changes, I need to build a new min.js file. So, how can I do it?

P.S. > I tried running the **npm run build** command. It successfully builds the min.js file. But, I get `' Module '"fabric"' has no exported member 'fabric''` error while I am running my App which uses `fabric as  import {fabric} from 'fabric';

Thanks
`

**Comment by ShaMan123** [comment]
It should work
You should provide more context
